packages/babel-template/src/index.js

```diff
@@ -99,6 +99,7 @@ let templateVisitor = {
   },
 
   exit({ node }) {
-    traverse.clearNode(node);
+    if (!node.file)
```

Comment:
I may have worked around the forOf error thing - shall I send a PR with the whitespace diffs then?